libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -908,6 +908,40 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 
 		break;
 	}
+	case FunctionType::Kind::Revert:
+	{
+		solAssert(arguments.size() == parameterTypes.size(), "");
+		if (arguments.empty())
+			m_code <<  "revert(0, 0)\n";
+		else
+			solAssert(false, "");
+
+		break;
+	}
+	case FunctionType::Kind::Send:
+	case FunctionType::Kind::Transfer:
+	{
+		solAssert(arguments.size() == 1 && parameterTypes.size() == 1, "");
+		string address{IRVariable(_functionCall.expression()).part("address").name()};
+		string value{expressionAsType(*arguments[0], *(parameterTypes[0]))};
+		Whiskers templ(R"(
+		let <success> := call(0, <address>, <value>, 0, 0, 0, 0)
```

Comment:
Please check ExpressionCompiler for a specific mechanism that sets the gas parameter to the stipend if the value is zero.